### ISSUE TITLE
Bower.json Main Section

### ISSUE THREAD

**Issue by mikecasas** [rdash/rdash-ui#12]
When using a gulp plugin to copy all my bower css files to a build folder, the rdash css gets copied twice, one as the regular css and one as the minified version. After looking at the bower documentation it seems that the main section should only have one file per filetype, but the package has two css files in that section.

http://bower.io/docs/creating-packages/#main

**Comment by Ehesp** [comment]
Yeah, to be fair I need to take another look at this project and re-do a lot of things. Just need the time.

**Comment by mikecasas** [comment]
No problem, I appreciate the project. I understand the time crunch.
